### ISSUE TITLE
[UI] Circle window of a Sonar Monitor overlaps its control panel

### ISSUE THREAD

**Issue by Singularia** [Regalis11/Barotrauma#11662]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Related to the issue mentioned in https://github.com/Regalis11/Barotrauma/issues/6856
Main circle of a Sonar Monitor is partially overlapping its control panel, which makes it's harder to use the control panel.

Possible fix is to change the current vanilla values of `relativesize` and `relativeoffset`
from
```xml
<GuiFrame relativesize="0.55,0.59" anchor="Center" style="OuterGlow" color="0,0,0,0.8" relativeoffset="0.1,-0.05" />
```
to 
```xml
<GuiFrame relativesize="0.63,0.60" anchor="Center" style="OuterGlow" color="0,0,0,0.8" relativeoffset="0.12,-0.05" />
```

Examples of visuals BEFORE and AFTER bellow
_(do not mind the alternative texture of the sonar screen)_

BEFORE
![image](https://user-images.githubusercontent.com/62324831/227766392-b2b81550-0226-4722-9a85-5d5d8417e524.png)

AFTER
![image](https://user-images.githubusercontent.com/62324831/227766405-ebfbd9ee-3efb-4848-b82b-1685c42a0342.png)

### Reproduction steps

In Sub-Editor or in SP/MP game take the submarine with dedicated Sonar Monitor and try to use controls of it with vanilla overlapping issue.

### Bug prevalence

Happens every time I play

### Version

v1.0.8.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I'm afraid I could not reproduce the issue. The circle does intentionally overlap the panel slightly, but it shouldn't go over the actual interactable elements.

I gave a test on various different resolutions and it seems to work correctly on all of them.

![image](https://user-images.githubusercontent.com/14351070/228854528-771d594e-9e52-44a2-a3ba-c1a71892b597.png)

What kind of resolution and UI scale settings are you using?

**Comment by Singularia** [comment]
HUD Scale: 120%
Inventory Scale: 100%
Text Scale: 110%
Resolution: 3840x2160
Display Mode: Borderless

Monitor/TV: Sony Bravia KDL-32w653a
Native resolution: 1920x1080
Display resolution: 3840x2160 (i use AMD's Virtual Super Resolution to archive this)
Windows scale is set to 200%

TL;DR:
1. The cause most likely is 120% HUD Scale
2. If you change HUD Scale nothing changes until you change your resolution.

My guess is that this is caused by the 120% HUD Scale.
(Interface with settings i mention at the start of the message)

![image](https://user-images.githubusercontent.com/62324831/229290346-f0326991-3112-47e5-9f5f-cddf07c4df69.png)


When I set HUD Scale to `100%` it didn't change a thing, but when i changed the Resolution and the Display Mode _(initially to 1920x1080/Windowed, but it can be any combination)_ it did.
When I changed the resolution and display mode back to what they were at the beginning, the interface stayed the same, no critical overlapping (Example below).

![image](https://user-images.githubusercontent.com/62324831/229289804-0949627a-60b1-4a69-9d99-126861473ef7.png)

P.S.
Why there is a need to make the intentional overlap in a first place?

**Comment by Regalis11** [comment]
Thanks for the extra info, I'll take another look at this with those settings!

> Why is there a need to make the intentional overlap in a first place?

The idea is just to make the UI look a little more compact: since the panel is at the top-right corner of the circle, where there's lots of empty space, there's no need to offset the panel completely outside the rectangular area of the sonar view.

**Comment by Regalis11** [comment]
Fixed in https://github.com/Regalis11/Barotrauma-development/commit/9a1c40e331086b2e46406295dd91491093cdbc0b

**Comment by Singularia** [comment]
> The idea is just to make the UI look a little more compact

Understandable, I guess, although I don't see why it must look compact, since the sonar monitor as an object is a kind of widescreen, if you ask me (a kind of dissonance here). 
Also, there is nothing on the screen other than: the sonar circle, the block of controls and the optionaly linked status monitor window (can't add anything else), so I see no problem with the first two elements spreading from each other a little.

IMO, the overlapping of the elements looks rather unattractive and is noticeable.

**Comment by Rokvach** [comment]
Tested, working correctly. Closing.
